app/javascript/components/Admin/MerchantAccounts/MerchantAccount.tsx

```diff
@@ -0,0 +1,139 @@
+import { Link } from "@inertiajs/react";
+import { capitalize } from "lodash";
+import React from "react";
+
+import DateTimeWithRelativeTooltip from "$app/components/Admin/DateTimeWithRelativeTooltip";
+import { BooleanIcon, NoIcon } from "$app/components/Admin/Icons";
+
+export type AdminMerchantAccountProps = {
+  id: number;
+  charge_processor_id: string;
+  charge_processor_merchant_id: string;
+  created_at: string;
+  external_id: string;
+  user_id: number;
+  country: string;
+  country_name: string;
+  currency: string;
+  holder_of_funds: string;
+  stripe_account_url: string;
+  charge_processor_alive_at: string;
+  charge_processor_verified_at: string;
+  charge_processor_deleted_at: string;
+  updated_at: string;
+  deleted_at: string;
+  live_attributes: Record<string, unknown>;
```

Comment:
Could we send this as an array instead? Would be nice to not have to use `Object.keys`